proto/mixer.proto

```diff
@@ -252,6 +252,43 @@ message GetPlacesInResponse {
   string payload = 1;
 }
 
+
+// Search request sent to the mixer.
+message SearchRequest {
+  // Search query which is just a plain text string, e.g. "california".
+  // This is not a datalog query, but a plain text query. The mixer is free
+  // to choose an interpretation of the query, e.g. using NLP or just plain
+  // keyword search and return relevant entities from the graph.
+  string query = 1;
+
+  // Maximum number of entities to return, default to 200.
+  int32 max_results = 2;
```

Comment:
This is maximum across all results? s.t. if total results exceed 200, then they are not reliable?